packages/fast-components-react-msft/package.json

```diff
@@ -88,6 +88,7 @@
     "babel-jest": "^23.6.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
+    "exenv-es6": "^1.0.0",
```

Comment:
I don't see this used elsewhere - is this necessary?

Reply:
Not any more - at one point I had to check the DOM. Will remove.